xbmc/input/InputManager.cpp

```diff
@@ -0,0 +1,406 @@
+/*
+*      Copyright (C) 2005-2014 Team XBMC
+*      http://xbmc.org
```

Comment:
being nitpicky: URL / copyright notice not up-to-date anymore :) (also on some other occasions)